go/tools/gazelle/rules/construct.go

```diff
@@ -145,6 +147,55 @@ func newValue(val interface{}) bf.Expr {
 	return nil
 }
 
+type byAttrName []keyvalue
+
+var _ sort.Interface = byAttrName{}
+
+func (s byAttrName) Len() int {
+	return len(s)
+}
+
+// namePriority is the sort order for rule attributes.
+// Copied from github.com/bazelbuild/buildifier/build/rewrite.go
+var namePriority = map[string]int{
```

Comment:
We already depend on buildifier, and NamePriority is a public symbol, why not use it directly rather than copy it?